css-positioning/relative.css

```diff
@@ -0,0 +1,41 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+  position: relative;
+  top: 10px;
+}
+
+#bulbasaur {
+  /* your code here */
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
+  top: 30px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 80px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 160px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 300px;
+}
```

Comment:
You already set `position: relative` in the `.pokemon > img` rule above, you don't need to set it again in each of these rules.